src/modules/mavlink/mavlink_receiver.h

```diff
@@ -234,7 +234,8 @@ class MavlinkReceiver : public ModuleParams
 	 *
 	 * @return PX4_OK on success, PX4_ERROR on fail.
 	 */
-	int set_message_interval(int msgId, float interval, int data_rate = -1);
+	int set_message_interval(int msgId, float interval, int data_rate = -1, float param3 = 0.0f, float param4 = 0.0f,
```

Comment:
Note, `response_target` has to be an int (in spec). `data_rate` is not in spec, param 5, 6 could be anything, but I'm restricting to ints here.